tests/conftest.py

```diff
@@ -20,6 +21,12 @@
 from pangeo_forge.storage import CacheFSSpecTarget, FSSpecTarget, UninitializedTarget
 
 
+def pytest_addoption(parser):
+    parser.addoption(
+        "--redirect-dask-worker-logs-to-stdout", action="store", default="NOTSET",
+    )
```

Comment:
This was a fun solution to https://github.com/pangeo-forge/pangeo-forge/issues/84#issuecomment-812075900.